### PR TITLE
Append scss extension

### PR DESCRIPTION
**Pull request by kylekatarnls**
Hi,

Here is what I had to do to be able to import scss files using a syntax like `~module-name/file` omitting `.scss` (for consistency with our other imports).

Thanks,

### CODE REVIEW

**Comment by kylekatarnls** [comment]
I tried many things but don't succeed to make jest working with Node.js 4.

**Comment by kylekatarnls** [comment]
@matthewdavidson @tadam313 I see 2 old PRs with no response, is this project abandoned?

**Comment by danoc** [comment]
Would love to see this merged. Happy to help out if I can.

**Comment by kylekatarnls** [comment]
I published it on npmjs.com, so you can require my fork meanwhile it's merged (if this project is not abandoned): `npm install --save @kylekatarnls/node-sass-tilde-importer`

File: index.test.js [hunk 1]
```diff
@@ -71,10 +79,20 @@ describe('Importer', function() {
...
+    mockFs.existsSync
+      .mockReturnValueOnce(false) // .scss import
+      .mockReturnValueOnce(true); // directory import
 
     expect(importer('~my-module/test', '')).toEqual({
       file: __dirname + '/MOCK_PARENT_DIR/node_modules/my-module/test/index'
     });
   });
+
+  test('should support directory imports', function() {
```
Comment (matthewdavidson) [inline index.test.js]:
```suggestion
  test('should support file imports minus extension', function() {
```

**Review by matthewdavidson — CHANGES_REQUESTED** [review]
Apologies for taking so long to review this. Overall this seems like a reasonable feature and in line with existing scss file resolution behaviour. Some small things to fix since I did some housekeeping, but otherwise LGTM.

**Comment by kylekatarnls** [comment]
Glad to hear! Changes applied.

**Review by matthewdavidson — APPROVED** [review]

**Comment by matthewdavidson** [comment]
Again, thanks @kylekatarnls 👏 

Released as an alpha as I don't actively use this module anymore so I'll rely on feedback from you folk before 2.0.0 release. Also gives us a chance to add more features in the mean time.

**Comment by danoc** [comment]
@matthewdavidson — I've verified that that `v2.0.0-alpha.1` fixes the problem for us. Would love to see an official `2.0.0` release if there is no additional in-flight work. Happy to help if there's anything I can do. 🙇

**Pull request merged**